components/automate-gateway/gateway/server.go

```diff
@@ -160,6 +163,10 @@ func (s *Server) Start() error {
 	return s.startSignalHandler()
 }
 
+func (s *Server) loadDatacollectorLimiter() {
+	s.datacollectorLimiter = limiter.NewInflightRequestLimiter("collector-requests", 800)
```

Comment:
I wonder if we want this to be something like 100*CpuCount

Reply:
maybe. the number is difficult to pick and the knobs that are now on ingest-service and compliance-service make it even harder.